Code/MinimalLib/jswrapper.cpp

```diff
@@ -20,6 +20,31 @@
 using namespace RDKit;
 
 namespace {
+class JSDrawerFromDetails : public MinimalLib::DrawerFromDetails {
+ public:
+  JSDrawerFromDetails(const emscripten::val &ctx, int w = -1, int h = -1,
+                      const std::string &details = std::string()) {
+    init(w, h, details);
+    d_ctx = ctx;
+  }
+
+ private:
+  MolDraw2DJS &drawer() const {
+    CHECK_INVARIANT(d_drawer, "d_drawer must not be null");
```

Comment:
Should be a PRECONDITION:
```suggestion
    PRECONDITION(d_drawer, "d_drawer must not be null");
```

Reply:
You are right; suggestion committed, thanks!